docs/getting-started/proofs-and-verification/curl/state.md

```diff
@@ -5,10 +5,10 @@ title: Proof of Existence of a State/Data
 
 ### State proof query
 
-As it mentioned in Orion description [here], Orion maintains separated persisted graph data structure for historical data transitions, so a user can execute query on those historical changes to understand the lineage of each data item. For more explanation about **Provenance Queries** and different views they provide on historical data, see [Provenance queries](../../queries/curl/provenance).
+As it mentioned in Orion description [here](../../../introduction) and detailed [here](../../../architecture-and-design/provenance-data), Orion maintains separated persisted graph data structure for historical data transitions, so a user can execute query on those historical changes to understand the lineage of each data item. For more explanation about **Provenance Queries** and different views they provide on historical data, see [Provenance queries](../../queries/curl/provenance).
 
-As complimentary to provenance, Orion uses Ethereum style Merkle-Particia Trie to provide cryptographically verifiable proofs of all state transitions. 
-Although it provides only one  single type of proof - that specific key was associated with specific value while specific block was committed to ledger.
+As complimentary to provenance, Orion uses Ethereum style Merkle-Patricia Trie to provide cryptographically verifiable proofs of all state transitions. 
```

Comment:
In addition to the provenance graph store, Orion uses an Ethereum style Merkle-Patricia Trie to provide cryptographically verifiable proofs of all state transitions. 
It provides only one single type of proof: that specific key was associated with specific value when a specific block was committed to ledger.